app/javascript/components/property-mapping-list/PropertyCard.jsx

```diff
@@ -7,6 +7,23 @@ import { propertyClassesForSpineTerm } from './stores/propertyMappingListStore';
  */
 const PropertyCard = ({ term }) => {
   const propertyClasses = propertyClassesForSpineTerm(term).map((c) => <li key={c}>{c}</li>);
+
+  const renderComments = (term) => {
```

Comment:
minor thing - would be nice to move this to some helper and re-use this at `PropertyAlignment`